### PR TITLE
FOGL-7476  Fixed opencv-contrib-python version.

### PR DESCRIPTION
**Pull request by deepanshu-yadav**
Used  **4.6.0.66**  last available version for python 3.6

### CODE REVIEW

**Comment by praveen-garg** [comment]
Don't we have the same issue in person-detector plugin (using bucket storage)?

**Comment by praveen-garg** [comment]
Also replace pip3 with python3 -m pip where required; as requested in Jira

**Comment by deepanshu-yadav** [comment]
> Don't we have the same issue in person-detector plugin (using bucket storage)?

Done

**Comment by deepanshu-yadav** [comment]
> Also replace pip3 with python3 -m pip where required; as requested in Jira

Done

File: extras_install.sh [hunk 1]
```diff
@@ -26,12 +26,12 @@ ID=$(cat /etc/os-release | grep -w ID | cut -f2 -d"=")
...
 if [ ${ID} = "raspbian" ]; then
-   pip3 install opencv-contrib-python==4.1.0.25
+   python3 -m pip  install opencv-contrib-python==4.1.0.25
 fi
 
 if [ ${ID} = "ubuntu" ]; then
-   pip3 install --upgrade pip
-   pip3 install opencv-contrib-python
+   python3 -m pip  install --upgrade pip
+   python3 -m pip  install opencv-contrib-python==4.6.0.66
```
Comment (ashish-jabble) [inline extras_install.sh:34]: Did you test with different python versions `<=3.6 & >=3.6`?
It's better to test against ub18 & ub20 platforms. On fresh instance & on upgraded/existing instance.

**Review by praveen-garg — APPROVED** [review]

**Review by AmandeepArora — APPROVED** [review]

**Pull request merged**